packages/core/src/execution-engine/workflow-execute.ts

```diff
@@ -986,6 +989,21 @@ export class WorkflowExecute {
 		return workflowIssues;
 	}
 
+	private getRoutingNodeExecuteFunction(node: INode, type: INodeType) {
```

Comment:
Shouldn't this be the opposite?

```suggestion
	private getNonRoutingNodeExecuteFunction(node: INode, type: INodeType) {
```

See also other comment on naming.